autofit/non_linear/paths/directory.py

```diff
@@ -348,9 +348,17 @@ def save_all(self, search_config_dict=None, info=None):
         VisualiseGraph(
             model=self.model,
         ).save(str(self.output_path / "model_graph.html"))
+
         if info:
             self.save_json("info", info)
+
         self.save_json("search", to_dict(self.search))
+        try:
+            info_start = self.search.initializer.info_from_model(model=self.model)
+            self._save_model_start_point(info=info_start)
+        except (NotImplementedError, AttributeError):
+            pass

```

Comment:
Could be worth logging something here as these errors may cover up code mistakes